robot/Cumulus/tests/browser/recurring_donations/zedit_effective_date_erd_record.robot

```diff
@@ -0,0 +1,127 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/RecurringDonationsPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             DateTime
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             Enable RD2
+#Suite Teardown  Delete Records and Close Browser
+
+*** Keywords ***
+
+Setup Test Data
+
+        ${effective_date_initial} =           Get Current Date      result_format=%-m/%-d/%Y
+        ${date} =           Get Current Date
+        ${DATE}=            Add Time To Date      ${date}  30 days      result_format=%Y-%m-%d %H:%M:%S.%f
+        ${date_to_update} =   Convert Date                   ${DATE}                         result_format=%Y-%m-%d
+        ${effective_modified_date}=        Add Time To Date      ${date}  0 days     result_format=%-m/%-d/%Y
+        ${currdate}=        Get Current Date  result_format=datetime
+        ${currentvalue} =   Evaluate          (${currdate.month-1}) * 100
+        ${currentvalue_edited}=   Evaluate    (${currdate.month}*100) + 150
+        Set Suite Variable  ${currentvalue}
+        Set Suite Variable  ${DATE}
+        Set Suite Variable  ${effective_modified_date}
+        Set Suite Variable  ${currentvalue_edited}
+        Set Suite Variable  ${effective_date_initial}
+        Set Suite Variable  ${date_to_update}
```

Comment:
Any variable being declared outside of the test needs to be in caps. This is advisable for all global variables.